### PR TITLE
Skipped tests list support added for TestableReference

### PR DESCRIPTION
**Pull request by somedev**
Added ability to work with `SkippedTests` array inside `TestableReference`

### CODE REVIEW

**Comment by segiddins** [comment]
@AliSoftware this look good to you?

**Comment by AliSoftware** [comment]
:+1: but don't forget to add the CHANGELOG entry

**Comment by somedev** [comment]
@segiddins @AliSoftware  CHANGELOG updated

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 ##### Enhancements
 
-* None.  
+* Add accessors for working with skipped tests inside TestAction  in `.xcscheme` files.  
+  [Eduard Panasiuk](https://github.com/somedev)
```
Comment (AliSoftware) [inline CHANGELOG.md]: Sorry to be pedantic, but could you also add the link to this PR too here? Thx!

Comment (somedev) [inline CHANGELOG.md]: No problem

**Comment by somedev** [comment]
CHANGELOG updated (added the link to this PR)

**Comment by AliSoftware** [comment]
Ace 👌 Thanks!

**Pull request merged**